themes/demo/layouts/default.htm

```diff
@@ -13,6 +13,7 @@
         <link rel="icon" type="image/png" href="{{ 'assets/images/winter.png'|theme }}">
         <link href="{{ 'assets/css/vendor.css'|theme }}" rel="stylesheet">
         <link href="{{ 'assets/css/theme.css'|theme }}" rel="stylesheet">
+        <link href="/modules/system/assets/css/framework.extras.css" rel="stylesheet">
```

Comment:
Should be unnecessary with the `{% snowboard %}` tag now right?

Reply:
No, the Snowboard tag only adds the JS. The user still needs to manually add the CSS. This is the same as the original framework.